hw/dv/_index.md

```diff
@@ -2,4 +2,4 @@
 title: "Design Verification"
 ---
 
-{{% sectionContent %}}
+{{% sectionContent type="section" depth="1" %}}
```

Comment:
Is the expectation to fill these sections with some description?

Reply:
Yes, I think that would be great.